airflow/www/api/experimental/endpoints.py

```diff
@@ -255,18 +413,14 @@ def dag_run_status(dag_id, execution_date):
         _log.info(error_message)
         response = jsonify({'error': error_message})
         response.status_code = 400
-
         return response
 
-    try:
-        info = get_dag_run_state(dag_id, execution_date)
-    except AirflowException as err:
-        _log.info(err)
-        response = jsonify(error="{}".format(err))
-        response.status_code = err.status_code
+    if not dagruns:
+        error_message = "No Dag run found with provided execution date"
+        response = jsonify(error="{}".format(error_message))
```

Comment:
Can simply this with just ```jsonify(error=error_message)```

Reply:
yea, this is done where ever it was possible to do so.